lib/bgs/vnp_relationships.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module BGS
+  class VnpRelationships
+    def initialize(proc_id:, veteran:, dependents:, user:)
+      @user = user
+      @veteran = veteran
+      @proc_id = proc_id
+      @dependents = dependents
+    end
+
+    def create
```

Comment:
Nitpicky - I'd probably name this `create_all` to make it a bit more obvious that we're creating _all_ the relationships.

Reply:
I've renamed this to `create_all`.